enterprise/models.py

```diff
@@ -2275,9 +2295,12 @@ def __str__(self):
         """
         Return human-readable string representation.
         """
-        return '<EnrollmentNotificationEmailTemplate for EnterpriseCustomer with UUID {}>'.format(
-            self.enterprise_customer.uuid
-        )
+        if self.enterprise_customer:
```

Comment:
If this is the string notification of the model object, please add the object's .id field so when you see this in logs you can actually go look at a database object.

Reply:
added! look like these in adjango admin 

```
<EnrollmentNotificationEmailTemplate (id: 5) Default template for type DEFAULT_ENROLL>
<EnrollmentNotificationEmailTemplate (id: 6) for EnterpriseCustomer with UUID 873fec72-5a21-4f95-ac19-e3d67f9c4777>
```